public/app/plugins/datasource/cloudwatch/aws_url.ts

```diff
@@ -12,7 +12,7 @@ export interface AwsUrl {
 }
 
 export function encodeUrl(obj: AwsUrl, region: string): string {
-  return `https://${region}.console.aws.amazon.com/cloudwatch/home?region=${region}#logsV2:logs-insights$3FqueryDetail$3D${JSURL.stringify(
+  return `https://${region}.console.aws.amazon.com/cloudwatch/home?region=${region}#logs-insights:queryDetail=${JSURL.stringify(
```

Comment:
What does the `logsV2` part means and why can we remove it here?

Reply:
logsV2 was originally there to link to the "new" cloudwatch logs insights interface, but I noticed the document didn't call for that to be used, so I changed it to use the original interface URL